roles/installer/ospd/overcloud/network/isolation/three-nics-vlans/tasks/loadbalancer.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: copy the loadbalancer template
+  template:
+      src: "external-lb.yaml.j2"
+      dest: "~/external-lb.yaml"
+
+- name: append the External LB template line to the base overcloud deploy script
+  lineinfile:
+      dest: "~/overcloud_deploy.sh"
+      line: '                       -e /usr/share/openstack-tripleo-heat-templates/environments/external-loadbalancer-vip.yaml \\'
```

Comment:
ansible 2 has removed the need for "\\", this will actually fail in ansible 2, please remove one of the "\".